### ISSUE TITLE
the --write option is verbose even if the file is not changed.

### ISSUE THREAD

**Issue by jfroffice** [prettier/prettier#3525]
Thank you for **prettier**, this is an amazing tool.

Is it possible to be less verbose when prettier write files?

I mean when the option `--write` is enable prettier rewrite file even it is not changing at all.

**Comment by ikatyang** [comment]
We did not rewrite formatted files, can you share steps to reproduce the issue?

https://github.com/prettier/prettier/blob/9780c003323c686be91cfa67d11eeb43bae1edeb/src/cli-util.js#L355-L375

**Comment by jfroffice** [comment]
```
prettier --tab-width 4 --single-quote --print-width 100 --trailing-comma es5 '{.,../..tests/acceptance}/**/*.{js,jsx}' --ignore-path ../../.gitignore --write

app/components/aaa1/aaa.js 60ms
app/components/aaa1/aaa2.js 27ms
app/components/aaa1/aaa3.js 6ms
app/components/aaa2/aaa.js 68ms
app/components/aaa2/aaa2.js 6ms
....
```

I probably misunderstood the log and the fact that the files were not overwritten.

I think it could be better if a **--silent** option to silence these logs.

**Comment by ikatyang** [comment]
FYI, the colors of formatted/unformatted files are different.

![image](https://user-images.githubusercontent.com/8341033/34159685-0fa23626-e505-11e7-8bc6-7bfc0f0f7713.png)

And we just merged #3515 yesterday 🎉 , which make all loggings respect `--loglevel` option.

**Comment by jfroffice** [comment]
@ikatyang thx. great!
